components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -121,18 +126,26 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 	}
 
 	// idempotent
+	l.Info("Migrating data if needed...")
```

Comment:
This is also a little confusing, but it helps if we were debugging. More and more I'm wondering if these should be set to debug by default.

Reply:
I think it is quite important to have them on by default, because if a customer runs into a problem we will at least have a clue what is going on. Asking them to re-run after setting to debug is not an adequate substitute.

Reply:
I agree the "if needed" is confusing--that is what could be alleviated by the change to `dataMigConf.Migrate()` I spoke of.

Reply:
what about "migrating any unmigrated data" or something (then it's always true)